sql/catalyst/src/test/scala/org/apache/spark/sql/UTF8StringBenchmark.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+import java.math.BigInteger
+import java.security.SecureRandom
+import java.util.Random
+
+import org.apache.spark.unsafe.types.UTF8String
+import org.apache.spark.util.Benchmark
+
+object UTF8StringBenchmark {
```

Comment:
Not sure what the convention is, but I don't know if we need to preserve this benchmarking code